docker/addons/vault/vault_copy_certs.sh

```diff
@@ -9,9 +9,21 @@ export MAGISTRALA_DIR=$scriptdir/../../../
 
 cd $scriptdir
 
+# Default .env file path
+env_file="$MAGISTRALA_DIR/docker/.env"
+
+# Parse command line arguments
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        --env-file) env_file="$2"; shift ;;
+        *) echo "Unknown parameter passed: $1"; exit 1 ;;
+    esac
+    shift
+done
+
 readDotEnv() {
```

Comment:
So something like this we can try 
```suggestion

# Default .env file path
env_file="docker/.env"

# Parse command line arguments
while [[ "$#" -gt 0 ]]; do
    case $1 in
        --env-file) env_file="$2"; shift ;;
        *) echo "Unknown parameter passed: $1"; exit 1 ;;
    esac
    shift
done

certs_copy_path="docker/ssl/certs/"
while [[ "$#" -gt 0 ]]; do
    case $1 in
        --dest-path) certs_copy_path="$2"; shift ;;
        *) echo "Unknown parameter passed: $1"; exit 1 ;;
    esac
    shift
done


readDotEnv() {
    set -o allexport
    source "$env_file"
    set +o allexport
}

readDotEnv

server_name="localhost"

# Check if MG_NGINX_SERVER_NAME is set or not empty
if [ -n "${MG_NGINX_SERVER_NAME:-}" ]; then
    server_name="$MG_NGINX_SERVER_NAME"
fi

echo "Copying certificate files"

if [ -e "data/${server_name}.crt" ]; then
    cp -v data/${server_name}.crt      ${certs_copy_path}/magistrala-server.crt
else
    echo "${server_name}.crt file not available"
fi

if [ -e "data/${server_name}.key" ]; then
    cp -v data/${server_name}.key      ${certs_copy_path}/magistrala-server.key
else
    echo "${server_name}.key file not available"
fi

if [ -e "data/${MG_VAULT_PKI_INT_FILE_NAME}.key" ]; then
    cp -v data/${MG_VAULT_PKI_INT_FILE_NAME}.key    ${certs_copy_path}/ca.key
else
    echo "data/${MG_VAULT_PKI_INT_FILE_NAME}.key file not available"
fi

if [ -e "data/${MG_VAULT_PKI_INT_FILE_NAME}_bundle.crt" ]; then
    cp -v data/${MG_VAULT_PKI_INT_FILE_NAME}_bundle.crt     ${certs_copy_path}/ca.crt
else
    echo "data/${MG_VAULT_PKI_INT_FILE_NAME}_bundle.crt file not available"
fi

exit 0

```

Reply:
[> So something like this we can try](https://github.com/absmach/magistrala/pull/2387#discussion_r1726947534)

After implementing this please check with https://github.com/absmach/amdm/pull/88 it's working

Reply:
@JeffMboya Please move all the scripts to a directory

Reply:
@JeffMboya 
@arvindh123 A quick clarification: move the scripts to a single directory inside the vault folder i.e `docker/addons/vault/scripts` or outside the vault folder i.e magistrala/scripts? https://github.com/absmach/magistrala/pull/2387#discussion_r1727039272


@Arvindh
Good question,
I think we can move to `docker/addons/vault/scripts` for now
What do you think?

@JeffMboya
I'm thinking of using `docker/addons/vault/scripts` because it keeps the scripts together with other Vault files.


@arvindh123
okay then lets go with `docker/addons/vault/scripts`

Reply:
All the vault scripts have been moved to `docker/addons/vault/scripts`